nntrainer/app_context.cpp

```diff
@@ -145,6 +145,19 @@ std::vector<std::string> getPluginPaths() {
   return ret;
 }
 
+/**
+ * @brief Get the Full Path from given string
+ * path is resolved in the following order
```

Comment:
move from here to the rest of the `@brief` description under `@details`?

Reply:
Done